backend/src/main/java/com/google/minicrm/data/Lead.java

```diff
@@ -89,7 +89,7 @@ public Lead(Entity entity) {
     this.isTest = (Boolean) entity.getProperty("isTest");
     this.adgroupId = (Long) entity.getProperty("adgroupId");
     this.creativeId = (Long) entity.getProperty("creativeId");
-    this.status = LeadStatus.values()[(int) entity.getProperty("status")];
+    this.status = LeadStatus.values()[((Long) entity.getProperty("status")).intValue()];
```

Comment:
does this not worth as the original int cast? Otherwise, does it work as just Integer?

Reply:
Okay, I think this is fine then. I think keeping as int is better because there are so few statuses.